packages/common-server/src/files.ts

```diff
@@ -70,6 +70,12 @@ export function readYAML(fpath: string): any {
   });
 }
 
+export async function readYAMLAsync(fpath: string): Promise<any> {
```

Comment:
nit: I think a better naming pattern is to have `readYAML` -> `readYAMLSync`.  The async version you can always tell by the return type.  But the Sync version with any an `any` return type can possibly be either sync or async in nature.  I'm guessing that's why fs-extra follows the pattern of calling out `sync` versions explicitly (and not the async versions). 

Maybe something for later since the other methods in this file already follow the `fooAsync` pattern.

Reply:
Agreed with the naming. I'll go ahead and rename those functions too, if it causes any conflicts I can fix them before merging.